### PR TITLE
Add icsd kaz

### PR DESCRIPTION
**Pull request by shenriod**
Add ICSD Kazakhstan to the main page of www.klink.asia

They wish to have links to the K-Box, the Website and the Microsite. Is this possible? If yes, did I do it correctly?

Thanks for checking that I have updated all necessary files!

### CODE REVIEW

File: source/index.md [hunk 1]
```diff
@@ -33,6 +33,12 @@ members:
   name: The **Global Project ValuES - Methods for Integrating Ecosystem Services into Policy, Planning, and Practice**, implemented by GIZ on behalf of BMU, as part of the German Government’s International Climate Initiative (IKI) 
   website: http://aboutvalues.net/
   background: values
+ icsd-kaz:
+  name: The **Interstate Commission on Sustainable Development (Scientific and Information Center Kazakhstan)** 
+  website: http://www.mkurca.org/
+  microsite: https://kz.icsd.klink.asia/projects/rpoosur/ru
```
Comment (avvertix) [inline source/index.md]: @shenriod the attribute `microsite` is not valid. Only `kbox` and `website` are used. The `kbox` attribute is usually used for navigating to the login page as a shortcut for users, while the `website` attribute is used to direct the visitor to the official description of the specific member. If you want to direct traffic to the microsite you could insert that URL into the `kbox` attribute.

Comment (shenriod) [reply source/index.md]: Done

**Review by avvertix — APPROVED** [review]

**Comment by shenriod** [comment]
Just so that I can inform the users, when can we expect a deploy to see the new version live? (no hurry)

**Comment by avvertix** [comment]
Usually it takes 15-20 minutes to complete the deploy procedure

**Pull request merged**